services/todo-legacy/repositories/todo.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  AnyObject,
+  DataSourceConstructor,
+  DefaultCrudRepository,
+  Filter,
+  Model,
+  ModelBaseConstructor,
+  ObjectType,
+  Repository,
+  Where,
+  juggler
+} from '@loopback/repository';
+import { inject, Application } from '@loopback/core';
+
+export class TodoRepository {
+  // FIXME(kev): The juggler.ModelBase type doesn't carry any definitions
+  // for the CRUD functions that typically get mixed in when createModel
+  // makes a definition, so we have to use type "any" to avoid complaint!
+  model: juggler.ModelBase | any;
+  datasource: juggler.DataSource;
+  constructor(
+    @inject('datasources.config') config: Object,
+    @inject('models.todo') modelDefinition: any
+  ) {
+    this.datasource = new DataSourceConstructor(config);
+    this.model = this.datasource.createModel(
+      modelDefinition.name,
+      modelDefinition.properties
+    );
+  }
+
+  // API definitions
+  async find(id?: number, filter?: Filter) {
+    return id ? await this.model.findById(id) : await this.model.find(filter);
```

Comment:
This isn't necessarily meant to be a pointless passthrough; we could do controller-specific control flow in these classes. I expect that there would be subsequent PRs to shore up this example as loopback-next's API becomes fixed in the coming weeks.